indico/modules/events/contributions/models/contributions.py

```diff
@@ -407,3 +410,19 @@ def _add_timetable_consistency_trigger(target, conn, **kw):
         EXECUTE PROCEDURE events.check_timetable_consistency('contribution');
     """.format(target.fullname)
     DDL(sql).execute(conn)
+
+
+@listens_for(Contribution, 'load')
+def _preload_principals(target, context):
```

Comment:
I like `*unused` to swallow extra arguments like `context` that aren't actually used. Like our `**kwargs` in Indico signal handlers.